src/main/resources/assets/alchemistry/lang/fr_fr.json

```diff
@@ -0,0 +1,66 @@
+{
+  "tooltip.alchemistry.evaporator": "Fonctionne %d plus lentement dans les biomes humides.",
+  "tooltip.alchemistry.energy_requirement":"Requiert %d FE/t.",
+
+  "block.alchemistry.chemical_dissolver": "Dissolveur chimique",
+  "block.alchemistry.chemical_combiner": "Mélangeur chimique",
+  "block.alchemistry.evaporator": "Évaporateur",
+  "block.alchemistry.atomizer": "Atomiseur",
+  "block.alchemistry.liquifier": "Liquéfacteur",
+  "block.alchemistry.fission_controller": "Contrôleur de chambre à fission",
+  "block.alchemistry.fission_core": "Cœur de chambre à fission",
+  "block.alchemistry.fission_casing": "Coffrage de chambre à fission",
+  "block.alchemistry.fusion_controller": "Contrôleur de chambre à fusion",
+  "block.alchemistry.fusion_core": "Cœur de chambre à fusion",
+  "block.alchemistry.fusion_casing": "Coffrage de chambre à fusion",
+
+  "item.alchemistry.mineral_salt": "Sel minéral",
+  "item.alchemistry.fertilizer": "Engrais",
+  "item.alchemistry.obsidian_breaker": "Brise-obsidienne",
```

Comment:
Casse-obsidienne

Reply:
> Casse-obsidienne
@Mazdallier

"Brise-obsidienne" is already an accurate translation of "Obsidian Breaker".
Why do you want it to be replaced with a synonym ? Is it different according to you ?